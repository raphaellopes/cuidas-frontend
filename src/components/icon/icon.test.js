// vendors
import React from 'react';
import renderer from 'react-test-renderer';

// locals
import Icon from './index';

const component = props => (renderer.create(
  <Icon {...props} />,
));

describe('<Icon />', () => {
  test('Should render the component', () => {
    const tree = component({ name: 'check' }).toJSON();
    expect(tree).toMatchSnapshot();
  });

  test('Should change the color when prop is set', () => {
    const tree = component({ name: 'check', color: 'primary' }).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
