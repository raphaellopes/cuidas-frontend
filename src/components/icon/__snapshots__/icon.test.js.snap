// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Icon /> Should change the color when prop is set 1`] = `
<i
  className="fa fa-check sc-bdVaJa fQTDTn"
  color="primary"
/>
`;

exports[`<Icon /> Should render the component 1`] = `
<i
  className="fa fa-check sc-bdVaJa jToUlD"
/>
`;
